SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.iOS/Crashes.cs

```diff
@@ -287,5 +287,10 @@ private static NSDictionary StringDictToNSDict(IDictionary<string, string> dict)
         {
             return NSDictionary.FromObjectsAndKeys(dict.Values.ToArray(), dict.Keys.ToArray());
         }
+
+        internal static void PlatformUnsetInstance()
```

Comment:
```suggestion
        private static void PlatformUnsetInstance()
```

The platform specific method is called only by shared code of the same partial class. Other classes must call the shared UnsetInstance that is internal.